students/christopher_gantt/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,10 @@
+for number in range(1,101):
+	
+	if number%5 == 0 and number%3 == 0:
```

Comment:
please correct the indentation, should be exactly 4 spaces (no tabs)


minor spacing issues that can be corrected with proper linter or Pycharm:

`if number % 5 == 0 and number % 3 == 0` `for number in range(1, 101)`

Reply:
Ok, fixed them.